cmd/dvotecli/commands/common.go

```diff
@@ -0,0 +1,26 @@
+package commands
+
+import (
+	"fmt"
+
+	"go.vocdoni.io/dvote/crypto/ethereum"
+)
+
+const MaxListIterations = 256
+
+var (
+	digested bool
+	signKey  *ethereum.SignKeys
+)
+
+func checkSignKey() error {
+	signKey = ethereum.NewSignKeys()
```

Comment:
Using signKey and privKey as global vars is not a good practice.  But in case you really need to have it globally declared, this function would need a refactor:
```
if privKey == "" { return fmt.Errorf("...") }
return signKey.AddHexKey(privKey)
```

Reply:
I declared a `options` type, and refactored it